Specs/DateTime/English/DateTimeModel.json

```diff
@@ -23005,5 +23005,35 @@
         }
       }
     ]
+  },
+  {
+    "Input": "I'll be out November 19-20",
+    "Context": {
+      "ReferenceDateTime": "2016-11-07T00:00:00"
+    },
+    "Results": [
+      {
+        "Text": "november 19-20",
+        "Start": 12,
+        "End": 25,
+        "TypeName": "datetimeV2.daterange",
+        "Resolution": {
+          "values": [
+            {
+              "timex": "(XXXX-11-19,XXXX-11-20,P1D)",
```

Comment:
Incorrect resolution. Text clearly implies P2D.

Reply:
Should we modify this behavior? 
But it will affect other existing test cases (like the ones pointed above)

Reply:
It would be great to add a flag if the last day must be included.

Because at this moment extra day is added for months and years (I will be off in April -> ends 1st of May)
But for weeks and days it’s not included (April 1-30 -> ends 30 Apr) 

while I (human) read both of these cases the same

Reply:
@tellarin, @venoua, should we address this in a separate issue? (since the parsing problem is not related to the bug solved here)

Reply:
date resolution is much bigger issue and should be solved in separated issue. please merge this PR

Reply:
@aitelint Yes, changes will come as part of another issue. Let me create one soon. We're discussing alternatives as it can be considered a breaking change.